tests/components/blink/test_config_flow.py

```diff
@@ -16,12 +19,10 @@ async def test_form(hass):
     assert result["errors"] == {}
 
     with patch(
-        "homeassistant.components.blink.config_flow.Blink",
-        return_value=Mock(
-            get_auth_token=Mock(return_value=True),
-            key_required=False,
-            login_response={},
-        ),
+        "homeassistant.components.blink.config_flow.Auth.startup", Mock()
```

Comment:
The default `new` is already a `MagicMock`. We probably don't need to specify `Mock`.